firebase_admin/_auth_utils.py

```diff
@@ -266,6 +266,15 @@ def validate_action_type(action_type):
             Valid values are {1}'.format(action_type, ', '.join(VALID_EMAIL_ACTION_TYPES)))
     return action_type
 
+def validate_provider_ids(provider_ids, required=False):
+    if provider_ids is None:
+        if required:
+            raise ValueError('Invalid provider IDs. The list must be non-empty.')
```

Comment:
This error message is not exactly correct. How about "Provider ID list must not be None".?

Reply:
I am changing the check to this so that it will check for empty list and None both 
```
    if not provider_ids:
        if required:
            raise ValueError('Invalid provider IDs. Provider ids should be provided')
        return []
```